cli/tests/testdata/init/main.ts

```diff
@@ -0,0 +1,3 @@
+if (import.meta.main) {
+  console.log("Log from main.ts that already exists")
+}
```

Comment:
@leighmcculloch actually, this file is unnecessary. Would you be able to remove it and instead just write out a file to the temp dir (ex. I think it's: `testdata_init_main_ts.write("");`)